Yoast/wordpress-seo
# Add margin on taxonomy settings for breadcrumbs

## Summary

This PR can be summarized in the following changelog entry:

* Adds margin below select fields so that there is space between taxonomy settings for breadcrumbs .

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* Go to Yoast SEO -> Search Appearance -> Breadcrumbs.
* Scroll down to "Taxonomy to show in breadcrumbs for content types".
* See space between vertically stacked select fields for content types.

## Quality assurance

* [x ] I have tested this code to the best of my abilities

Fixes #10161
